concepts/commerce/checkout-concept/cart.md

```diff
@@ -105,7 +140,28 @@ A default set of collectors is implemented in Shopware 6, which has a set call o
 | Shopware\Core\Checkout\Promotion\Cart\CartPromotionsCollector | Enrich add, remove and validate promotions |
 | Shopware\Core\Checkout\Shipping\Cart\ShippingMethodPriceCollector | Handle shipping prices |
 
-![Cart enrichment steps](../../../assets/cart-enrichtment-steps.png)
+```mermaid
```

Comment:
It looks compared to the old diagram, like the *Collect* step is missing

Reply:
I was asked to remove it by checkout team